taqo_client/lib/widgets/app_drawer.dart

```diff
@@ -69,6 +69,26 @@ class _TaqoAppDrawerWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final authProvider = Provider.of<AuthProvider>(context);
 
+    // Added for bug #72
```

Comment:
Please remove this comment, and also later `// End of add for bug #72`. I think some comments that more explicitly explains the code might be better.

Reply:
Made the changes.